dev_tools/requirements/deps/protos.txt

```diff
@@ -3,6 +3,6 @@
 # 1.26.0 is the min version that has python 3.8 wheels
 # however, we can't really go much higher - the protoc version needs to be <= 3.9.2
 # otherwise it will generate pb2 files that are incompatible with tensorflow quantum
-grpcio-tools~=1.26.0
+grpcio-tools~=1.34.1
```

Comment:
We will need to update the preceding comment to indicate the actual state of the version constraints, since we're apparently no longer limited to 1.26.

Reply:
Added a comment, but leaving unresolved until Michael can weigh in.